### ISSUE TITLE
Links to Schema files dont' work

### ISSUE THREAD

**Issue by jag3773** [bible-technology/scripture-burrito#245]
Links like https://burrito.bible/schema/id_authorities.schema.json as found on https://docs.burrito.bible/en/v0.3.0-alpha.2/schema_docs/id_authorities.html don't work.

This may be due to how the schema files are generated or referenced?

We could either alias from the main domain or just link directly to the github raw endpoint. Not sure what's the best route, but making them active would help @jtauber .

**Comment by jtauber** [comment]
<https://burrito.bible/schema/id_authorities.schema.json> is actually the `$id` of the schema file, not just a link so I think we need further discussion. We should discuss whether we want to keep the `$id` in this form and then potentially work out how to alias / redirect.

**Comment by jag3773** [comment]
Yes, understand @jtauber -- we want it to function, one way or another :)

**Comment by jag3773** [comment]
Discover whether these links/ids should be versioned or permalinks.

**Comment by jag3773** [comment]
Loop in with #247

**Comment by jtauber** [comment]
@jag3773 can we add a DNS entry for `burrito.bible` to `ALIAS` to `bible-technology.github.io` (preferred) OR add `A` records:
```
185.199.108.153
185.199.109.153
185.199.110.153
185.199.111.153
```
